scripts/docker/ubuntu.16.04/Dockerfile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+FROM ubuntu:16.04
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like)
```

Comment:
@eerhardt As much as possible I think we should keep the entire definition of the dockerfile self-contained despite duplication. It makes things much more predictable and less complex.